cluster/gce/addons/konnectivity-agent/daemonset.yaml

```diff
@@ -28,14 +28,12 @@ spec:
           hostPath:
             path: /etc/srv/kubernetes/pki/konnectivity-agent
       containers:
-        - image: gcr.io/google-containers/proxy-agent:v0.0.3
+        - image: gcr.io/jying-gke-dev/proxy-agent-amd64:842202cc3b819fc307b46e2b92bd8a4391bb9231
           name: konnectivity-agent
           command: ["/proxy-agent"]
           args: [
                   "--logtostderr=true",
-                  "--ca-cert=/etc/srv/kubernetes/pki/konnectivity-agent/ca.crt",
-                  "--agent-cert=/etc/srv/kubernetes/pki/konnectivity-agent/client.crt",
-                  "--agent-key=/etc/srv/kubernetes/pki/konnectivity-agent/client.key",
```

Comment:
Given that network proxy is still in alpha and we're planning to remove certs anyway, I feel like it's fine to remove this and add the token validation in a later PR. Otherwise, we'd still have to partially generate certs and mount proper volumes only to have them be removed a PR later.